apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -150,10 +150,10 @@ export class VersionAction extends BaseRushAction {
         const newPolicyVersion: semver.SemVer = new semver.SemVer(policy.version);
         if (newPolicyVersion.prerelease.length) {
           // Update 1.5.0-alpha.10 to 1.5.0-beta.10
-          newPolicyVersion.prerelease[0] = this._prereleaseIdentifier.value;
+          newPolicyVersion.prerelease = [ this._prereleaseIdentifier.value, ...newPolicyVersion.prerelease.slice(1) ];
```

Comment:
I didn't understand this change, or how it relates to PNPM workspaces.

Reply:
Bumping the semver types caused this to fail, as the type is now a readonly array. By doing this, we're setting the array to a newly constructed array with the value at the beginning, and slices off the first in the original array, in order to maintain the same functionality with the new type

Reply:
```suggestion
          // For example, if we are parsing "1.5.0-alpha.10" then the newPolicyVersion.prerelease array
          // would contain [ "alpha", 10 ], so we would replace "alpha" with "beta"
          newPolicyVersion.prerelease = [ this._prereleaseIdentifier.value, ...newPolicyVersion.prerelease.slice(1) ];
```